flake.nix

```diff
@@ -83,6 +83,20 @@
           '';
 
         };
+
+        packages.pulse-exe = pkgs.writeShellScriptBin "pulse.exe" ''
+          exec ${inputs.fstar.packages.${system}.fstar}/bin/fstar.exe "$1" \
+            --include ${config.packages.pulse}/lib/pulse \
+            --include ${config.packages.pulse}/lib/pulse/c \
+            --include ${config.packages.pulse}/lib/pulse/core \
+            --include ${config.packages.pulse}/lib/pulse/lib \
+            --include ${config.packages.pulse}/lib/pulse/lib/class \
+            --include ${config.packages.pulse}/lib/pulse/lib/ml \
+            --include ${config.packages.pulse}/lib/pulse/lib/pledge \
+            --load_cmxs pulse \
+            "$@"
+        '';
```

Comment:
A nit here, given that `lib/fstar.include` exists, the only thing needed is `--include ${config.packages.pulse}/lib`.

(Also F* will try to automatically load plugins when there's is a `#lang-X` directive, so the `--load_cmxs` is *mostly* redundant, but still required for some files currently, so best to keep it.)

Reply:
OK, I have to figure out how to get the `lib/fstar.include` to be found. For some reason I could not get the command to work unless the paths are exhaustively and explicitly included.

Reply:
Ah.. interesting. Then this is fine and we can later try to reduce it, fstar.include is rather new so no worries.